test/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnerAddressControllerSpec.scala

```diff
@@ -71,12 +71,12 @@ class BusinessPartnerAddressControllerSpec extends ControllerSpecWithGuiceApp wi
         setupUserAction()
         when(mockAppConfig.newBusinessPartnerPagesEnabled).thenReturn(true)
         val request = FakeRequest()
-          .withFormUrlEncodedBody(("partnerPostcode", "AB1 2YZ"))
-        val result = await(csrfAddToken(controller.load())(request))
+          .withFormUrlEncodedBody(("businessPartnersAddressPostcode", "AB1 2YZ"))
+          .withMethod("POST")
+        val result = await(csrfAddToken(controller.next())(request))
 
         status(result) shouldBe OK
-        val page = Jsoup.parse(contentAsString(result))
-        page.title should include("What is the partner’s address?")
```

Comment:
There is a test missing here for when the address line is supplied

Reply:
Test added.